lib/actions/attendees.actions.ts

```
@@ -0,0 +1,23 @@
+  userId: string;
+}) => {
+  const idParam = prepareIdParam(userId);
+  try {
+    const checkAssistedQuery = transformQuery("SELECT assisted from attendees WHERE id = ?");
+    const [assistedRows] = await db.query(checkAssistedQuery, [idParam]);
+    const assistedResult: boolean = assistedRows[0]?.assisted;
+    const updateAssistedQuery = transformQuery(
+      "UPDATE attendees SET assisted = ? WHERE id = ?"
+    );
+    const [attendeeRows] = await db.query(updateAssistedQuery, [!assistedResult, idParam]);
+    const [attendeeRows] = await db.query(updateAssistedQuery, [idParam]);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'attendeeRows'. Consider to delete it or rename it.

'attendeeRows' is defined here:


(lint/suspicious/noRedeclare)